### PR TITLE
Add method get_team_membership for user 

### PR DESCRIPTION
**Pull request by talmachani**
Implemented the get_team_membership with tests.

### CODE REVIEW

File: github/Team.pyi [hunk 1]
```diff
@@ -17,6 +18,7 @@ class Team(CompletableGithubObject):
         self, member: NamedUser, role: Union[str, _NotSetType] = ...
     ) -> None: ...
     def add_to_members(self, member: NamedUser) -> None: ...
+    def get_team_membership(self, member) -> Membership: ...
```
Comment (s-t-e-v-e-n-k) [inline github/Team.pyi:21]: You need to specify the type of member too.

File: tests/Team.py [hunk 2]
```diff
@@ -105,6 +105,20 @@ def testMembers(self):
         self.assertRaises(AssertionError, self.team.add_membership, user, "admin")
         self.team.remove_membership(user)
 
+    def testTeamMembership(self):
+        user = self.g.get_user("jacquev6")
+        self.assertListKeyEqual(self.team.get_members(), None, [])
+        self.assertFalse(self.team.has_in_members(user))
+        self.team.add_to_members(user)
```
Comment (s-t-e-v-e-n-k) [inline tests/Team.py:112]: add_to_members() API call is deprecated. Use `add_membership` instead.

File: tests/Team.py [hunk 3]
```diff
@@ -105,6 +105,20 @@ def testMembers(self):
         self.assertRaises(AssertionError, self.team.add_membership, user, "admin")
         self.team.remove_membership(user)
 
+    def testTeamMembership(self):
+        user = self.g.get_user("jacquev6")
+        self.assertListKeyEqual(self.team.get_members(), None, [])
```
Comment (s-t-e-v-e-n-k) [inline tests/Team.py:110]: This feels awkward, just:

``self.assertEqual(list(self.team.get_members()), [])``

File: github/Team.pyi (hunk 1 above)
Comment (talmachani) [reply github/Team.pyi]: Add Union[str,NamedUser]

File: tests/Team.py (hunk 2 above)
Comment (talmachani) [reply tests/Team.py]: changed both in test and ReplayData

File: tests/Team.py (hunk 3 above)
Comment (talmachani) [reply tests/Team.py]: done.

**Pull request merged**